staging/src/k8s.io/client-go/tools/leaderelection/resourcelock/interface.go

```diff
@@ -190,21 +174,15 @@ func New(lockType string, ns string, name string, coreClient corev1.CoreV1Interf
 	}
 	switch lockType {
 	case endpointsResourceLock:
-		return nil, fmt.Errorf("endpoints lock is removed, migrate to %s", EndpointsLeasesResourceLock)
+		return nil, fmt.Errorf("endpoints lock is removed, migrate to %s", endpointsLeasesResourceLock)
 	case configMapsResourceLock:
-		return nil, fmt.Errorf("configmaps lock is removed, migrate to %s", ConfigMapsLeasesResourceLock)
+		return nil, fmt.Errorf("configmaps lock is removed, migrate to %s", configMapsLeasesResourceLock)
```

Comment:
this message now points people at something unsupported by the current client-go version... maybe include the recommended client-go version for this in this message (v0.27.x)?

Reply:
done - PTAL